src/main/java/frc/robot/sensors/ElevatorLight.java

```diff
@@ -103,4 +118,12 @@ public LEDPattern greenProgress(DoubleSupplier progress) {
     LEDPattern heightDisplay = base.mask(mask);
     return heightDisplay;
   }
+
+  public Trigger thirtySecondsLeft() {
```

Comment:
This isn't inherently tied to the LEDs and should instead go in `Controls.java()`.